CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
-## Unreleased
+## [2.5.0.0.4]
```

Comment:
Wanna fill this in? ;)

Reply:
sure can! i'll move this to draft for now

Reply:
5c718ffbfeebbf6c5ac4cdbb534abd3e15f7cd96